slasher/service/data_update.go

```diff
@@ -37,3 +46,134 @@ func (s *Service) finalisedChangeUpdater() error {
 		}
 	}
 }
+
+func (s *Service) slasherOldAtetstationFeeder() error {
+	if s.beaconClient == nil {
+		return fmt.Errorf("can't feed old attestations to slasher. beacon client has not been started")
+	}
+	ch, err := s.beaconClient.GetChainHead(s.context, &ptypes.Empty{})
+	if err != nil {
+		log.Error(err)
+		return err
+	}
+	if ch.FinalizedEpoch < 2 {
+		log.Info("archive node doesnt have historic data for slasher to proccess. finalized epoch: %d", ch.FinalizedEpoch)
+	}
+	log.Infof("Finalized epoch is currently: %d", ch.FinalizedEpoch)
```

Comment:
This needs better logging for our log parsing utils. Typically, we display values as `log.WithField("finalizedEpoch", ch.FinalizedEpoch).Info("Performing some operation")` using camelCase and the WithField method from logrus. For multiple values in the field, use [WithFields](https://godoc.org/github.com/sirupsen/logrus#WithFields)

Reply:
changed. thanks